neural_compressor/keras/algorithms/static_quantize.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import Dict
+
+import tensorflow as tf
+
+from neural_compressor import quantization
+from neural_compressor.adaptor.torch_utils.util import fetch_module, set_module
```

Comment:
Please note that the `neural-compressor[tf]` binary only includes the `neural_compressor/tensorflow` and `neural_compressor/common`. Importing components outside of these directories is not allowed.
But, it's okay with me if it's used to reduce duplicate work and removed later by PR.